CHATOPS.md

```diff
@@ -0,0 +1,33 @@
+
+### NOTE: Development work on ChatOps has been suspended
+
+# Aliases and Chatops
```

Comment:
Can we drop this document? I don't see us using chatops since it has some security and implementation considerations

Reply:
removed